src/network/api_implementation.py

```diff
@@ -72,89 +71,45 @@ def check_status_code(self, response):
         # l'eccezione è intercettata dal gruppo ServerErrs
         response.raise_for_status()
 
-    def cookie2str(self, cookie: dict) -> str:
-        for key, value in cookie.items():
-            if key == "ZM_AUTH_TOKEN":
-                return "ZM_AUTH_TOKEN=" + value
-        return ""
-
     @ExceptionsHandler
     def is_logged(self, _cookie: str = "") -> bool:
```

Comment:
Perchè in questo metodo non settiamo `self.cookie`? Non settandolo la prossima call di rete che fa va a richiedere un nuovo cookie